src/calendareventlistmodel.cpp

```diff
@@ -49,9 +49,10 @@ CalendarEventListModel::CalendarEventListModel(QObject *parent)
 
 CalendarEventListModel::~CalendarEventListModel()
 {
-    CalendarManager::instance()->cancelEventListRefresh(this);
+    if (CalendarManager::instance(false)) {
+        CalendarManager::instance(false)->cancelEventListRefresh(this);
```

Comment:
Doesn't matter much, but could store the instance into variable to avoid fetching twice.

Reply:
Right, added an intermediate variable.